spack/packages/dla-future/package.py

```diff
@@ -16,23 +15,41 @@ class DlaFuture(CMakePackage):
 
     version('develop', branch='master')
 
-    variant('gpu', default=False,
+    variant('cuda', default=False,
             description='Use the GPU/cuBLAS back end.')
-
-    # Until mpich is default comment this out
+    variant('test', default=False,
+            description='Run unit tests.')
+    variant('doc', default=False,
+            description='Build documentation.')
+    
     #depends_on('mpi@3:')
-    depends_on('mpich')
-    depends_on('mkl')
+    depends_on('mpi')
+    depends_on('blas')
+    depends_on('lapack')
     depends_on('blaspp')
     depends_on('lapackpp')
-    depends_on('hpx cxxstd=14 networking=none')
-    depends_on('cuda', when='gpu=True')
+    depends_on('hpx@1.4.0 cxxstd=14 networking=none')
```

Comment:
This stick us to `1.4.0`, but I would say that we require **AT LEAST** 1.4.0.

I know that something is going on with the HPX API, so maybe the check may become a bit more complicated than simply 1.4.0+.

@teonnik have a better overview than me on this.

Reply:
> I know that something is going on with the HPX API, so maybe the check may become a bit more complicated than simply 1.4.0+.

1.4.0+ should be OK for now. When HPX 1.5 is released, we will bump that to 1.5.0. So the line can be changed to :
`depends_on('hpx@1.4.0: cxxstd=14 networking=none')`